pkg/function/backup_data.go

```diff
@@ -60,6 +60,8 @@ const (
 	BackupDataOutputBackupSize = "size"
 	// BackupDataOutputBackupPhysicalSize is the key used for returning physical size taken by the snapshot
 	BackupDataOutputBackupPhysicalSize = "phySize"
+	// InsecureTLS is the key name which provides an option to a user to disable tls
+	InsecureTLS = "insecureTLS"
```

Comment:
```suggestion
	// BackupDataInsecureTLS is the key name which provides an option to a user to disable tls
	BackupDataInsecureTLS = "insecureTLS"
```
To make it consistent with other declarations

Reply:
I think since it's common in various functions. So let's keep it as it is. WDYT @PrasadG193 ?

Reply:
ok, makese sense